package.json

```diff
@@ -33,6 +33,7 @@
         "plugins/deploy-git",
         "plugins/deploy-git/example",
         "plugins/path-to-regexp",
+        "plugins/koa",
```

Comment:
We need to add `plugins/koa/example` as well

Reply:
Done